Subject: Missing #include <assert.h>

From : Anatoli Koutsevol <Anatoli.Koutsevol@MarkovProcesses.com>

===================================================================
RCS file: /cvsroot/boomerang/boomerang/loader/BinaryFile.cpp,v
retrieving revision 1.6
diff -u -r1.6 BinaryFile.cpp
--- loader/BinaryFile.cpp	30 Jan 2004 11:27:47 -0000	1.6
+++ loader/BinaryFile.cpp	1 Feb 2004 00:59:04 -0000
@@ -41,6 +41,7 @@
 #include "HpSomBinaryFile.h"
 #include "ExeBinaryFile.h"
 #include <stdio.h>
+#include <assert.h>
 #ifndef WIN32
 #include <dlfcn.h>
 #endif
